handler_test.go

```diff
@@ -160,7 +160,7 @@ func TestHandler_ClusterResizeAbort(t *testing.T) {
 	t.Run("No resize job", func(t *testing.T) {
 		h := test.NewHandler()
 		h.API.Cluster = test.NewCluster(1)
-		h.SetRestricted()
+		h.API.Cluster.SetState(pilosa.ClusterStateResizing)
```

Comment:
whoa, gross - Cluster should not be exported on API. Probably, this test should be moved into api_test.go and be in `package pilosa` so that it can set the unexported state.

Reply:
I suggest we make this a new ticket.

Reply:
yep - will do